blivet/size.py

```diff
@@ -269,6 +269,9 @@ def __repr__(self):
     def __deepcopy__(self, memo):
         return Size(self.convertTo())
 
+    def __reduce__(self):
+        return (self.__class__, (str(self.convertTo()),))
```

Comment:
I think this is worth a comment about the meaning  of the ``__reduce__`` method and it's return type/format. Without it I think many people would have to study the docs like I did which is quite cumbersome. Also the returned value looks really weird if the return type is not mentioned/described.